src/main/groovy/edu/oregonstate/mist/locations/resources/LocationResource.groovy

```diff
@@ -24,29 +25,33 @@ import javax.ws.rs.core.Response
 
 @Path("/locations")
 @Produces(MediaType.APPLICATION_JSON)
+@PermitAll
 class LocationResource extends Resource {
     private final CampusMapLocationDAO campusMapLocationDAO
     private final DiningDAO diningDAO
     private final LocationDAO locationDAO
     private final ExtensionDAO extensionDAO
     private final ArcGisDAO arcGisDAO
+    private final CulCenterDAO culCenterDAO
+    private ExtraData extraData
 
     LocationResource(CampusMapLocationDAO campusMapLocationDAO,
-                     DiningDAO diningDAO,
-                     LocationDAO locationDAO,
-                     ExtensionDAO extensionDAO,
-                     ArcGisDAO arcGisDAO) {
+                     DiningDAO diningDAO, LocationDAO locationDAO,
+                     ExtensionDAO extensionDAO, ArcGisDAO arcGisDAO, CulCenterDAO culCenterDAO,
+                     ExtraData extraData) {
         this.campusMapLocationDAO = campusMapLocationDAO
         this.diningDAO = diningDAO
         this.locationDAO = locationDAO
         this.extensionDAO = extensionDAO
         this.arcGisDAO = arcGisDAO
+        this.culCenterDAO = culCenterDAO
+        this.extraData = extraData
     }
 
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     @Timed
-    Response getCampusMap(@Auth AuthenticatedUser authenticatedUser) {
+    Response getCampusMap() {
```

Comment:
Is there any point of hanging on to this endpoint and related DAO? It returns a 500 ever since we moved away from using the database and instead get data from campusmap.json